### ISSUE TITLE
Assemblies written by Mono.Cecil implicitly claim to have been built by VS2005

### ISSUE THREAD

**Issue by SteveGilham** [jbevain/cecil#485]
This got spotted by running a scan for places where old tools were still being used in production, which picked up Cecil-modified artifacts as one class of false-positive.

If I build a C# assembly with VS 2017, running `dumpbin /headers` gives (amongst a few screens-full of other data)
```
OPTIONAL HEADER VALUES
             10B magic # (PE32)
           48.00 linker version
```
Running the dumpbin tool on the result of loading that assembly into Cecil and then writing it out to a new location will show
```
OPTIONAL HEADER VALUES
             10B magic # (PE32)
            8.00 linker version
```
which is the linker version for VS 2005.

**Comment by jbevain** [comment]
Hello Steve,

Thanks for reporting this. Do we really care? I imagine we could preserve the linker version, but this has no impact on the viability of the resulting assembly.

**Comment by SteveGilham** [comment]
The requirement is, I agree, somewhat recherché; but it is one of those things that, if it becomes an issue is an issue.  In the particular case I have encountered this, a security audit based on checking for code displaying known-insecure linker versions triggered on Cecil-rewritten assemblies.

The simplest possible technical resolution would be not to claim any linker version at all e.g. by zeroing out the field entirely thus -- [patch.zip](https://github.com/jbevain/cecil/files/1686031/patch.zip); but copying the version across for rewrites, rather than ground-up construction, is also reasonable.  It will take me a little longer to put that together.

Related [https://github.com/Microsoft/visualfsharp/issues/4291](https://github.com/Microsoft/visualfsharp/issues/4291)

**Comment by jbevain** [comment]
Fixed in #486.
